### PR TITLE
Added stars related instructions for MoskitoAsesinato

### LINKED ISSUE #168: Number of stars in mosquito assasinato needed to buy a new product
## Description
As a [USER],
I need know how many stars are needed to buy the health boost and spray,
so that I can buy accordingly.

### Update [Required]
- [ ] Add a functionality using which the user can know how many stars are needed to buy a thing. 

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentos.

## Estimation
1-2 hours

### PR DESCRIPTION
**Pull request by lunayach**
### Description
Added instructions related to spending of stars to boost spray/health in MosktioAsesinato game.

Fixes #168 

### Type of Change:

- Code

### How Has This Been Tested?
All existing Mocha and Karma tests passed.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/app/unlocked-stage/unlocked-stage.component.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @import "../styles/main";
 
+$gif-height: 500px;
```
Comment (Raxerz) [inline src/app/unlocked-stage/unlocked-stage.component.scss]: Is this responsive?
Better to have % units

**Comment by Raxerz** [comment]
Fix the conflict @lunayach

File: src/assets/languages/english.json [hunk 2]
```diff
@@ -392,8 +392,10 @@
           "heading": "Prevent the mosquito from transmitting the virus to the human",
           "instructions": {
             "title": "How to play?",
-            "ins1": "Select the inventory item",
-            "ins2": "Use the item to stop the mosquito"
+            "ins1": "Select the inventory item.",
+            "ins2": "Use the item to stop the mosquito.",
```
[the remark this answers is not in the document]
Comment (lunayach) [reply src/assets/languages/english.json]: Done.

File: src/app/unlocked-stage/unlocked-stage.component.scss (hunk 1 above)
Comment (lunayach) [reply src/app/unlocked-stage/unlocked-stage.component.scss]: **%** was not working properly. I converted into **em**'s which is also a relative unit and would be responsive.

**Comment by Raxerz** [comment]
@shreyans29 please check if this PR looks fine so that we can merge it.

**Pull request merged**